specs-jetpack-calypso/wp-jetpack-connect-spec.js

```diff
@@ -223,18 +223,17 @@ test.describe( `Jetpack Connect: (${ screenSize })`, function() {
 			return driverManager.ensureNotLoggedIn( driver );
 		} );
 
-		test.it( 'Can register new Subscriber user', function() {
-			this.blogName = dataHelper.getNewBlogName();
-			this.emailAddress = dataHelper.getEmailAddress( this.blogName, signupInboxId );
+		test.it( 'Can register new Subscriber user', () => {
```

Comment:
Is there any reason for that change? It doesn't go inline with our style guide

Reply:
Nope, fixed 👍